Javascript-2/Week-1/Movie App/index.html

```diff
@@ -0,0 +1,200 @@
+    />
+    <script type="module" src="script.js" defer></script>
+    <script src="arrayList.js"></script>
+</head>
+<body>
+    <header>
+        <!-- <img
+          src="https://cdn2.iconfinder.com/data/icons/video-36/100/video-film-roll-video-player-media-app-film-roll-movie-512.png"
+          ,
+          width="100"
+          ,
+          height="100"
+        /> -->
+
+        <p class="site-name"><i class="fa fa-video-camera col_red me-1" style="margin-right: 10px"></i>Flimy Fly</p>
+  
+        <!-- <nav class="nav">
+          <ul>
+            <li><a href="">Home</a></li>
+            <li><a href="">Movies</a></li>
+            <li><a href="">Series</a></li>
+          </ul>
+        </nav> -->
+        <div class="forUser">
+          <!-- <span>
+            <i id="shorting" , class="fa-solid fa-list"></i>
+            <fieldset id="filters">
+              <p id="newRelease" class="filter">New Releases</p>
+              <p id="actors" class="filter">Actors</p>
+              <p id="releaseYear" class="filter">Release Year</p>
+              <label
+                ><select id="dropdown">
+                  <option>select one</option>
+                  <option>1970-1979</option>
+                  <option>1980-1989</option>
+                  <option>1990-1999</option>
+                  <option>2000-2009</option>
+                  <option>2010-2019</option>
+                  <option>2020-Till Now</option>
+                </select></label
+              >
+              <p id="genres" ,class="filter">genres</p>
+            </fieldset>
+          </span> -->
+          <!-- <span> <i class="fa-solid fa-bell"></i></span> -->
+          <span>
+            <input
+              type="search"
+              ,
+              id="keyword-search"
+              ,
+              placeholder="search"
+              required 
+              style="width: 300px; height: 40px;"
+            />
+            <i id="search-icon" class="fa-solid fa-magnifying-glass"></i>
+          </span>
+          <span>
+            <div id="loginButton"><a href="">Login</a></div>
+            <fieldset id="logIn">
+              <label
+                ><input
+                  type="email"
+                  ,
+                  id="email"
+                  ,
+                  placeholder="Enter Email Address"
+              /></label>
+              <label
+                ><input
+                  type="password"
+                  ,
+                  id="password"
+                  ,
+                  minlength="8"
+                  ,
+                  placeholder="Your password"
+                  ,
+                  pattern
+                  required /></label
+              ><br />
+              <button id="signIn">Sign In</button>
+              <p>If not registered! Please Sign up</p>
+              <button id="signUp">Sign Up</button><br />
+              <div id="signUpPage">
+                <label
```

Comment:
```suggestion
                <label>
                Enter Your First Name:
```
Please keep the closing `>` on the same line as the tag name.